rust/agama-lib/src/store.rs

```diff
@@ -71,6 +74,7 @@ impl Store {
     pub async fn load(&self) -> Result<InstallSettings, ServiceError> {
         let mut settings = InstallSettings {
             bootloader: Some(self.bootloader.load().await?),
+            hostname: Some(self.hostname.load().await?),
```

Comment:
store part is missing...that means if you do `agama config load` with hostname, then `agama config show` will miss it ( I know it as I did identical mistake :)